.github/workflows/main.yaml

```diff
@@ -23,24 +9,45 @@ on:
       - 'master'
       - 'staging'
       - 'trying'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
1. I'm concerned that this ends in `+*` which is uncommon in regex syntax (I would expect it to mean ((one or more of the preceding) zero or more of the preceding).

2. Optional: consider namespacing the release tag names, like `release/1.2.3`. 

Is this a full match or a partial match?

Reply:
Yeah, I agree this is bad, perhaps it needs a comment, but this isn't regex... it's a thing that looks like regex. `*` means  `[^/]+` (or maybe `[^/]*`) here I think 🤦

Reply:
I actually can't express what I actually want (which I express in the grep command for the VERSION var) because it's not expressive enough, so I had to just replace the entire last part with `*`

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet